v2/car.go

```diff
@@ -137,3 +137,6 @@ func (h *Header) ReadFrom(r io.Reader) (int64, error) {
 	h.IndexOffset = binary.LittleEndian.Uint64(buf[16:])
 	return n, nil
 }
+
+// TODO: Need a utility to get a CARv2 file from a CARv1 file, basically a CARv1 -> indexed CARv2 transformer func.
```

Comment:
I'll add a high-level API for this, which would wrap an existing carv1 as a carv2 while keeping the carv1 bytes identical and building an index.

Reply:
Now tracked in https://github.com/ipld/go-car/issues/119.